src/main/kotlin/com/kryszak/gwatlin/api/mapinfo/GWMapInfoClient.kt

```diff
@@ -22,19 +20,21 @@ class GWMapInfoClient(
      * Returns a list of [com.kryszak.gwatlin.api.mapinfo.model.Map]
      * instances corresponding to the specified IDs from the /maps endpoint
      * @param mapIds a collection of map IDs
-     * @param language an ISO 639-1 language code, defaults to the language specified when constructing this instance
+     * @param language one of the languages defined in [ApiLanguage], or null to use the default language specified
+     * when creating this client, if any
```

Comment:
Part `to use the default language specified
     * when creating this client, if any` does not longer apply after removal of default language on client itself